docs/swagger/traPortfolio.v1.yaml

```
@@ -0,0 +1,168 @@
+openapi: 3.0.0
+info:
+  title: traPortfolio
+  version: '1.0'
+servers:
+  - url: 'http://localhost:3000'
+paths:
+  /users:
+    get:
+      summary: ユーザーのリストを取得
+      tags: []
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/User'
+      operationId: getUsers
+      description: |-
+        ユーザー情報を取得します
+        `include-suspended`を指定しない場合、レスポンスに非アクティブユーザーは含まれません。
+      parameters:
+        - schema:
+            type: boolean
+            default: 'false'
+          in: query
+          name: include-suspended
+          description: アカウントがアクティブでないユーザーを含めるかどうか
+  '/users/{userId}':
+    parameters:
+      - $ref: '#/components/parameters/userIdInPath'
+    get:
+      summary: ユーザー詳細情報の取得
+      tags: []
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/UserDetail'
+        '404':
+          description: Not Found
+      operationId: getUser
+      description: ユーザー詳細情報を取得します
+    patch:
+      summary: ユーザー情報の編集
+      operationId: edithUser
+      responses:
+        '200':
+          description: OK
+        '400':
+          description: Bad Request
+        '404':
+          description: Not Found
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/PostUserRequest'
```

```suggestion
              $ref: '#/components/schemas/EditUserRequest'
```
`EditUserRequest`の構造体下に定義してほしい:pray:

---

ここpatchいらない気がした

---

ないと「自己紹介文」と「氏名を公開するかどうかの可否」を変更できるエンドポイントないと思う

---

確かにそうだった🙏